projects/javascript/To-do List.md

```diff
@@ -0,0 +1,5 @@
+- Creating a To-do List
```

Comment:
only need one idea, so pick your "best" for now

Reply:
I picked one of the javascript projects for the contribution project and attached the link.